src/SDKs/Batch/DataPlane/Azure.Batch.IntegrationTests/App.config

```diff
@@ -7,7 +7,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
```

Comment:
@matthchr .NET SDK that targets .NET Core is at newtonsoft >=9.0
But .NET 452 is still targeting 6.0.8. This is mainly for powershell constraint.